drivers/hmis/app/graphql/types/hmis_schema/ce_referral_event.rb

```diff
@@ -0,0 +1,38 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::CeReferralEvent < Types::BaseObject
+    # object is a Hmis::WorkflowExecution::AuditEvent
+    field :id, ID, null: false
+    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :user, Types::Application::User, null: true
+    field :type, String, null: false, method: :event_type
```

Comment:
```suggestion
    field :type, String, null: false
```
type is defined below